src/components/file-picker/.eslintrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I think we can define the overrides in the top-level eslintrc with a files regex.

Reply:
Done in 68ee789718372a7284ceb21a8dbdfefd5e8eb268